packages/kotti-ui/source/kotti-table/KtTable.vue

```diff
@@ -336,14 +442,12 @@ export default {
 	text-align: center;
 }
 
-::v-deep .kt-table tr.disabled {
-	cursor: not-allowed;
-	background-color: #f8f8f8;
-	opacity: 0.6;
-}
-
 .x-scroll {
```

Comment:
for other reviewers:
this was moved from here https://github.com/3YOURMIND/kotti/pull/787/files#diff-604842dfb9282c755ebaa023a8363c33fdb4c010b18bff8f8170c49e41acf633L103